packages/react/src/utils/dom.ts

```diff
@@ -76,3 +77,25 @@ export async function sendCommand<T = string, R = unknown>(frame: HTMLIFrameElem
     frame.contentWindow?.postMessage(command, new URL(frame.src).origin, [channel.port2]);
   });
 }
+
+/**
+ * Creates a Blob object from the JSON object given and downloads the object.
+ * @param jsonString - The JSON string.
+ * @param fileName - Optional file name. Default is based on current timestamp.
+ */
+export function exportJsonFile(jsonString: string, fileName?: string): void {
```

Comment:
Huh, didn't know we had this!

There's a very similar method in `packages/react/src/PatientExportForm/PatientExportForm.tsx` - mental note that we can de-dupe.